python/desc/sims/GCRCatSimInterface/twinklesCatalogDefs.py

```diff
@@ -0,0 +1,79 @@
+"""PhoSim Instance Catalog"""
```

Comment:
Why can't you import this straight from desc.twinkles? I saw you mentioned this replication in your comment. Did I change something you needed in my Twinkles update and can we edit what you need in the Twinkles repo instead?

Reply:
Actually, I made things indpendent. Had meant to delete those. Will delete them.